keyboards/handwired/phantagom/gigan/keymaps/default/config.h

```diff
@@ -0,0 +1,4 @@
+// Copyright 2023 <dennis@kruyt.org>
```

Comment:
This file should be removed

Reply:
why, i need the DYNAMIC_KEYMAP_LAYER_COUNT?

Reply:
Not for the default keymap, where VIA (and by extension, dynamic keymaps) is not enabled.